src/core/openthread.cpp

```diff
@@ -1636,6 +1636,21 @@ ThreadError otSetActiveDataset(otInstance *aInstance, const otOperationalDataset
     return error;
 }
 
+bool otIsNodeCommissioned(otInstance *aInstance)
+{
+    otOperationalDataset aDataset;
```

Comment:
nit: `aDataset` -> `dataset` (the `a` prefix on variables should be used to indicate arguments).